Dockerfile

```
@@ -1,14 +1,17 @@
-FROM golang:1.20
+FROM ubuntu:latest
 
-RUN apt-get update && apt-get install -y vim git
-RUN mkdir -p /usr/src/gosh
-WORKDIR /usr/src/gosh
+RUN apt-get update && apt-get install -y vim git && apt install sudo
```

[ASK]
質問です！sudoってinstallする必要ありますか🤔？
最初から使えるものと思ってるのでinstallしてるのが気になりました！

---

あと、ここに下のやつもまとめちゃいましょう！もしsudoがいらなければこんな感じだと思います！
```suggestion
RUN apt-get update && apt-get install vim git software-properties-common -y
```

---

build時にsudo command not foundみたいなエラーが出ていたのでインストールしました
ただsudoは必要なかったのでsudoのインストールとsudoコマンドは全て削除しました！